### PR TITLE
[PB-3315]: feat/update seats in business sub

### PR DESCRIPTION
**Pull request by xabg2**
This PR adds functionality to update seat counts for existing business subscriptions. Organizations can now increase or decrease seats based on their needs.

### CODE REVIEW

File: src/controller/payments.controller.ts [hunk 1]
```diff
@@ -16,25 +16,26 @@ import {
...
+import fastifyJwt from '@fastify/jwt';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const rateLimit = require('fastify-rate-limit');
 
 type AllowedMethods = 'GET' | 'POST';
 
 export const allowedCurrency = ['eur', 'usd'];
 
-const allowedRoutes: {
+const ALLOWED_DOMAINS: {
```
Comment (sg-gs) [inline src/controller/payments.controller.ts:38]: These are allowedPaths or routes, these are not domains

File: src/controller/business.controller.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          additionalOptions: {
+            proration_behavior: 'create_prorations',
+          },
+        });
+
+        await usersService.updateWorkspaceStorage(user.uuid, Number(maxSpaceBytes), workspaceUpdatedSeats);
+
+        return updatedSub;
+      } catch (error) {
+        //
```
Comment (sg-gs) [inline src/controller/business.controller.ts:69]: Remove this comment and handle the error properly (log + appropriate response)

File: src/controller/business.controller.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      try {
+        await request.jwtVerify();
+      } catch (err) {
+        request.log.warn(`JWT verification failed with error: ${(err as Error).message}`);
+        reply.status(401).send();
+      }
+    });
+
+    fastify.patch<{ Body: { workspaceUpdatedSeats: number } }>('/subscription', async (req, res) => {
+      const { workspaceUpdatedSeats } = req.body;
```
Comment (sg-gs) [inline src/controller/business.controller.ts:28]: Validate that this is an integer and not something else.

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/controller/payments.controller.ts (hunk 1 above)
Comment (xabg2) [reply src/controller/payments.controller.ts]: Name updated.

File: src/controller/business.controller.ts (hunk 2 above)
Comment (xabg2) [reply src/controller/business.controller.ts]: Error handled properly

**Review by sg-gs — APPROVED** [review]

**Pull request merged**